meinberlin/apps/dashboard/dashboard.py

```diff
@@ -93,5 +94,49 @@ def get_progress(self, project):
         return num_valid + phase_num_valid, num_required + phase_num_required
 
 
+class ParticipantsComponent(DashboardComponent):
```

Comment:
i'd move the participants/moderator components and views to the same package as the accept logic.
this could either be the dashboard or the project. but i prefer the project to keep the dashboard tight